<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPM</class>
 <widget class="QDialog" name="CPM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>431</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>129</width>
     <height>124</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Set Background</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X_spin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_1">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Y_spin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_Set_Background">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>11</y>
     <width>166</width>
     <height>124</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Set Cell Model</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBox_Single_Cell_Locx">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBox_Single_Cell_Locy">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_Single_Cell_R">
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>N</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spinBox_Cell_N">
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Initialization</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>301</width>
     <height>154</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Set Parameters</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBox_time">
      <property name="maximum">
       <number>100000000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_T">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Chemical force</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Mechanical force</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Check Parameters</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Set Cell Parameters</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="0" column="1">
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Volume Co</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Peri Co</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
